src/librbd/cache/pwl/LogEntry.h

```diff
@@ -217,7 +217,7 @@ class WriteLogEntry : public GenericWriteLogEntry {
   virtual buffer::list &get_cache_bl() = 0;
 
   BlockExtent block_extent();
-  unsigned int reader_count() const;
+  virtual unsigned int reader_count() const;
```

Comment:
I would make it pure virtual and move the existing implementation to `rwl/WriteLog.cc`.

Reply:
done